src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -44,7 +44,7 @@ services:
   app:
     # on the verge of every deployment (fortnightly), update the "image" value from the "app" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/versions/RELEASE.json
-    image: daschswiss/dsp-app:v11.22.6
+    image: daschswiss/dsp-app:v11.25.0
     volumes:
```

Comment:
Hey @irmastnt , you forgot to bump the version of ingest. The latest version is 0.18.1, but here, we still have 0.17.0